src/ForgettingWeights.jl

```diff
@@ -0,0 +1,24 @@
+module ForgettingWeights
+
+using Compat
+
+function forgetting_weights(N::Int, lf::Int)::Vector{Float64}
+    if N < 0 throw(ArgumentError("forgetting_weights: $(N) below 0")) end
+
+    if N < lf
+        return ones(N)
+    end
+
+    ramp = @compat range(0., stop = 1., length = N - lf + 2)[2:end-1]
```

Comment:
Do we have any view on how an actual linear ramp changes modelling performance?

Reply:
There's been a comparison of modelling performance added to the `compare_fweigths\` folder together with any supporting files needed to replicate the experiment and there seems not to be any substantial difference in performance after the ramp changes.